solutions/standard/variables.tf

```diff
@@ -354,3 +354,15 @@ variable "elasticsearch_full_version" {
   type        = string
   default     = null
 }
+
+variable "image_port" {
```

Comment:
you should use `kibana` in the variable name so its clear what port this is for

Reply:
Done